calico_rkt/constants.py

```diff
@@ -33,4 +33,8 @@
 
 # CNI Constants
 CNI_CMD_ADD = "ADD"
-CNI_CMD_DELETE = "DEL"
\ No newline at end of file
+CNI_CMD_DELETE = "DEL"
+
+# Logging Configuration
+LOG_DIR = "/var/log/calico/cni"
+LOG_FILENAME = "cni.log"
```

Comment:
I don't think the filename should be common.

The CNI plugin should have a separate log from the IPAM plugin.  The IPAM plugin can emit logs to its own logfile, as well as to `stderr`, and then the CNI plugin can capture the `stderr` output from the IPAM plugin and log it out into its own log file.